receiver/mongodbatlasreceiver/internal/mongodb_atlas_client.go

```diff
@@ -553,3 +555,33 @@ func (s *MongoDBAtlasClient) processDiskMeasurementsPage(
 	}
 	return measurements.Measurements, hasNext(measurements.Links), nil
 }
+
+func (s *MongoDBAtlasClient) GetLogs(ctx context.Context, groupID, hostname, logName string, col_interval time.Duration) (*bytes.Buffer, error) {
+	buf := bytes.NewBuffer([]byte{})
+
+	t := strconv.Itoa(int(time.Now().Add(-col_interval).Unix()))
```

Comment:
This seems like it might drop log records occasionally, I think it would make more sense to keep track of what the last EndDate you passed was and use that as the next StartDate instead.